[2024-10-13T19:31:34.073+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-13T19:31:34.081+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-13T19:31:25.423958+00:00 [queued]>
[2024-10-13T19:31:34.085+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-13T19:31:25.423958+00:00 [queued]>
[2024-10-13T19:31:34.086+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-13T19:31:34.096+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_dim_customer_table> on 2024-10-13 19:31:25.423958+00:00
[2024-10-13T19:31:34.101+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9317) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-13T19:31:34.101+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'csv_to_star_schema_to_s3_to_redshift', 'create_dim_customer_table', 'manual__2024-10-13T19:31:25.423958+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/sales_pipeline.py', '--cfg-path', '/tmp/tmp31zoiwqf']
[2024-10-13T19:31:34.104+0000] {standard_task_runner.py:72} INFO - Started process 9325 to run task
[2024-10-13T19:31:34.104+0000] {standard_task_runner.py:105} INFO - Job 210: Subtask create_dim_customer_table
[2024-10-13T19:31:34.118+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-13T19:31:34.138+0000] {task_command.py:467} INFO - Running <TaskInstance: csv_to_star_schema_to_s3_to_redshift.create_dim_customer_table manual__2024-10-13T19:31:25.423958+00:00 [running]> on host 66045655c750
[2024-10-13T19:31:34.195+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_to_star_schema_to_s3_to_redshift' AIRFLOW_CTX_TASK_ID='create_dim_customer_table' AIRFLOW_CTX_EXECUTION_DATE='2024-10-13T19:31:25.423958+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-13T19:31:25.423958+00:00'
[2024-10-13T19:31:34.196+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-13T19:31:34.204+0000] {sql.py:266} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS etl_data.dim_customer (
            Customer_Id INT, Gender VARCHAR(255), Customer_Login_type VARCHAR(255), customer_key INT
        );
        
[2024-10-13T19:31:34.211+0000] {base.py:84} INFO - Retrieving connection 'redshift_default'
[2024-10-13T19:31:34.216+0000] {base.py:84} INFO - Retrieving connection 'redshift_default'
[2024-10-13T19:31:34.220+0000] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-10-13T19:31:34.221+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-10-13T19:31:35.446+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-10-13T19:31:38.492+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS etl_data.dim_customer (
            Customer_Id INT, Gender VARCHAR(255), Customer_Login_type VARCHAR(255), customer_key INT
        );
        , parameters: None
[2024-10-13T19:31:39.759+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-13T19:31:39.760+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=csv_to_star_schema_to_s3_to_redshift, task_id=create_dim_customer_table, run_id=manual__2024-10-13T19:31:25.423958+00:00, execution_date=20241013T193125, start_date=20241013T193134, end_date=20241013T193139
[2024-10-13T19:31:39.804+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-13T19:31:39.816+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-13T19:31:39.820+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
